src/Client/MainComponents/Widgets.fs

```diff
@@ -210,71 +208,32 @@ type Widget =
-                    prop.children [
-                        SidebarComponents.SidebarLayout.LogicContainer [
-                            Html.div [
-                                SelectiveTemplateFromDBModal.ToProtocolSearchElement model dispatch
-                            ]
-                            if model.ProtocolState.TemplateSelected.IsSome then                    
-                                Html.div [
-                                    SelectiveImportModal.RadioPluginsBox(
-                                        "Import Type",
-                                        "fa-solid fa-cog",
-                                        importTypeState.ImportType,
-                                        "importType ",
-                                        [|
-                                            ARCtrl.TableJoinOptions.Headers, " Column Headers";
-                                            ARCtrl.TableJoinOptions.WithUnit, " ..With Units";
-                                            ARCtrl.TableJoinOptions.WithValues, " ..With Values";
-                                        |],
-                                        fun importType -> {importTypeState with ImportType = importType} |> setImportTypeState
-                                    )
-                                ]
-                                Html.div [
-                                    ModalElements.Box(
-                                        "Rename Table",
-                                        "fa-solid fa-cog",
-                                        SelectiveTemplateFromDBModal.CheckBoxForTakeOverTemplateName(useTemplateName, setUseTemplateName, model.ProtocolState.TemplateSelected.Value.Name))
-                                ]
-                                Html.div [
-                                    ModalElements.Box(
-                                        model.ProtocolState.TemplateSelected.Value.Name,
-                                        "",
-                                        SelectiveTemplateFromDBModal.displaySelectedProtocolElements(model, selectedColumns, setSelectedColumns, dispatch, false))
-                                ]
-                            Html.div [
-                                SelectiveTemplateFromDBModal.AddFromDBToTableButton model selectedColumns importTypeState useTemplateName.TemplateName dispatch
-                            ]
-                        ]
-                    ]
+                    prop.className "flex flex-col gap-2"
+                    prop.children (SelectiveTemplateFromDB.Main(model, dispatch))
                 ]
             ]
+
         let content =
-            let switchContent = if model.ProtocolState.TemplateSelected.IsNone then selectContent() else insertContent()
+            let switchContent =
+                if model.ProtocolState.TemplatesSelected.Length > 0 && model.PageState.SidebarPage = Routing.SidebarPage.Protocol then
```

Comment:
`model.PageState.SidebarPage = Routing.SidebarPage.Protocol` we are still in the widget? Why check for sidebar pagestate?

Reply:
Otherwise, the view for the protocolsearch is left immediately, when a single template is added, without the possibility to add additional templates.

Reply:
Then you must adjust the Msg that triggers this.